### PR TITLE
pages/*: reword descriptions without using "a CLI for" etc.

### PR DESCRIPTION
**Pull request by acuteenvy**

### CODE REVIEW

File: pages/common/emulator.md [hunk 1]
```diff
@@ -34,3 +30,7 @@
 - Start an emulator with a given SD card partition image file:
 
 `emulator -avd {{name}} -sdcard {{path/to/sdcard.img}}`
+
+- Display the help:
```
Comment (kbdharun) [inline pages/common/emulator.md:34]:
```suggestion
- Display help:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. Just a minor suggestion and the PR is GTG.

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
@acuteenvy Is this PR GTG?

If there are still some pages to update feel free to mark this as a draft

> We could rebase and merge the commits if required.

**Comment by acuteenvy** [comment]
There are still a lot of pages to update but I don't have that much time right now to work on this, I'll make a new PR later.
Let's merge this as it is for now.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**